src/sentry/snuba/discover.py

```diff
@@ -1657,31 +1657,36 @@ def find_span_histogram_min_max(span, min_value, max_value, user_query, params,
     :param {str: str} params: Filtering parameters with start, end, project_id, environment
     :param str data_filter: Indicate the filter strategy to be applied to the data.
     """
-
     if min_value is not None and max_value is not None:
         return min_value, max_value
 
     selected_columns = []
     min_column = ""
     max_column = ""
+    q1_column = ""
+    q3_column = ""
```

Comment:
You and I both know these are quartiles, and I know it's from existing, so if you are going to keep this, I'd probably make it a bit more explicit for the future people working on this.
```
quantile25_column
quantile75_column
```
Or maybe name them by their function
```
outlier_lower_fence
outlier_upper_fence
```
Up to you though, it's minor.

Reply:
+1 for more descriptive names. renamed the variables 👍